net.waterfox.waterfox.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: net.waterfox.waterfox
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: waterfox
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
```

Comment:
does it support wayland or not?

Reply:
Left over from doing some testing, should support wayland.

Reply:
then if one has only X11, this will fail.